### PR TITLE
Bump docker-compose to 1.28.0

### PR DESCRIPTION
**Pull request by aiordache**
Post-release docker-compose 1.28.0

### CODE REVIEW

File: compose/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+## 1.28.0 
+(2021-01-20)
+
+### Features
+
+- Added support for Nvidia GPUs via device requests.
+
+- Added support for service profiles.
+
+- Changed the SSH connection approach to the Docker CLI's via shellout to the local SSH client (old behaviour enabled by setting `COMPOSE_PARAMIKO_SSH` environment variable).
```
Comment (usha-mandya) [inline compose/release-notes.md:19]:
```suggestion
- Changed the SSH connection approach to the Docker CLI by shelling out to the local SSH client. You can enable the old behaviour by setting `COMPOSE_PARAMIKO_SSH` environment variable.
```

Comment (usha-mandya) [reply compose/release-notes.md:19]: @aiordache Let me know if this makes sense?

File: compose/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,69 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+## 1.28.0 
+(2021-01-20)
+
+### Features
+
+- Added support for Nvidia GPUs via device requests.
```
Comment (usha-mandya) [inline compose/release-notes.md:15]:
```suggestion
- Added support for NVIDIA GPUs through device requests.
```

File: compose/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Added support for service profiles.
+
+- Changed the SSH connection approach to the Docker CLI's via shellout to the local SSH client (old behaviour enabled by setting `COMPOSE_PARAMIKO_SSH` environment variable).
+
+- Added flag to disable log prefix.
```
Comment (usha-mandya) [inline compose/release-notes.md:21]:
```suggestion
- Added a flag to disable log prefix.
```

File: compose/release-notes.md [hunk 4]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Added flag for ansi output control.
```
Comment (usha-mandya) [inline compose/release-notes.md:23]:
```suggestion
- Added a flag for ANSI output control.
```

File: compose/release-notes.md [hunk 5]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+### Bugs
+
+- Made `parallel_pull=True` by default.
+
+- Restored the warning for configs in non-swarm mode.
+
+- Took `--file` into account when defining `project_dir`.
+
+- Fixed service attach bug on `compose up`.
```
Comment (usha-mandya) [inline compose/release-notes.md:33]:
```suggestion
- Fixed a service attach bug on `compose up`.
```

File: compose/release-notes.md [hunk 6]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+### Miscellaneous
+
+- Made COMPOSE_DOCKER_CLI_BUILD=1 the default.
```
Comment (usha-mandya) [inline compose/release-notes.md:37]:
```suggestion
- Made `COMPOSE_DOCKER_CLI_BUILD=1` the default.
```

File: compose/release-notes.md [hunk 7]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Added usage metrics.
+
+- Synced schema with COMPOSE specification.
+
+- Improved failure report for missing mandatory environment variables.
+
+- Bumped attrs to 20.3.0.
```
Comment (usha-mandya) [inline compose/release-notes.md:45]:
```suggestion
- Bumped `attrs` to 20.3.0.
```

File: compose/release-notes.md [hunk 8]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped more_itertools to 8.6.0.
```
Comment (usha-mandya) [inline compose/release-notes.md:47]:
```suggestion
- Bumped `more_itertools` to 8.6.0.
```

File: compose/release-notes.md [hunk 9]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped cryptograhy to 3.2.1.
+
+- Bumped cffi to 1.14.4.
```
Comment (usha-mandya) [inline compose/release-notes.md:51]:
```suggestion
- Bumped `cffi` to 1.14.4.
```

File: compose/release-notes.md [hunk 10]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped virtualenv to 20.2.2.
```
Comment (usha-mandya) [inline compose/release-notes.md:53]:
```suggestion
- Bumped `virtualenv` to 20.2.2.
```

File: compose/release-notes.md [hunk 11]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped bcrypt to 3.2.0.
```
Comment (usha-mandya) [inline compose/release-notes.md:55]:
```suggestion
- Bumped `bcrypt` to 3.2.0.
```

File: compose/release-notes.md [hunk 12]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped gitpython to 3.1.11.
```
Comment (usha-mandya) [inline compose/release-notes.md:57]:
```suggestion
- Bumped GitPython to 3.1.11.
```

File: compose/release-notes.md [hunk 13]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped docker-py to 4.4.1.
+
+- Bumped Python to 3.9.
+
+- Linux: bumped Debian base image from stretch to buster (required for Python 3.9).
+
+- macOS: OpenSSL 1.1.1g to 1.1.1h, Python 3.7.7 to 3.9.0.
```
Comment (usha-mandya) [inline compose/release-notes.md:65]:
```suggestion
- macOS: Bumped OpenSSL 1.1.1g to 1.1.1h, and Python 3.7.7 to 3.9.0.
```

File: compose/release-notes.md [hunk 14]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Bumped pyinstaller 4.1.
```
Comment (usha-mandya) [inline compose/release-notes.md:67]:
```suggestion
- Bumped PyInstaller to 4.1.
```

File: compose/release-notes.md [hunk 15]
```diff
@@ -7,6 +7,69 @@ redirect_from:
...
+
+- Loosed restriction on base images to latest minor.
```
Comment (usha-mandya) [inline compose/release-notes.md:69]:
```suggestion
- Relaxed the restriction on base images to latest minor.
```

**Comment by usha-mandya** [comment]
@aiordache Thanks for the PR. Looks really good. I've suggested some minor updates. PTAL and let me know if you have any questions.

File: compose/release-notes.md (hunk 1 above)
Comment (thaJeztah) [reply compose/release-notes.md:19]: Does the `COMPOSE_PARAMIKO_SSH` need to contain a specific value (e.g. `COMPOSE_PARAMIKO_SSH=1`)?

Perhaps use active voice for the second sentence;

```suggestion
- Changed the SSH connection approach to the Docker CLI by shelling out to the local SSH client. Set the `COMPOSE_PARAMIKO_SSH` environment variable to enable the old behavior.
```

File: compose/release-notes.md (hunk 6 above)
Comment (thaJeztah) [reply compose/release-notes.md:37]: Perhaps move this to "features" (not sure). I moved https://github.com/docker/docker.github.io/pull/11858 out of "draft" (which is related to this change).

```suggestion
- Docker Compose now uses the native Docker CLI's `build` command when building images. Set the `COMPOSE_DOCKER_CLI_BUILD=0` environment variable to disable this feature.
```

**Comment by aiordache** [comment]
@usha-mandya, @thaJeztah Thank you for reviewing!

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**